src/utils/diff/parsers/common.ts

```diff
@@ -149,17 +184,30 @@ export abstract class BaseParser {
         return { added, removed }
     }
 
-    match(content: string): RegExpExecArray | null {
-        return this.buildRegexPattern().exec(content)
+    match(content: string): MatchResult | null {
+        const pattern = this.buildRegexPattern()
+
+        const matches = pattern.exec(content)
+        if (matches) {
+            const varName = matches[1] || matches[2]
+            return {
+                isUnknown: !varName.match(/^["'].*["']$/),
+                // position or named parameter match
+                name: varName.replace(/["']/g, ''),
```

Comment:
```suggestion
                name: varName.replace(/^['"]|['"]$/g, ''),
```
what about something like this? to only replace quotes at the start or end of the string